native/src/modules/categories/components/CategoryListItem.js

```diff
@@ -49,9 +49,9 @@ const CategoryTitle = styled(Highlighter)`
 const CategoryThumbnail = styled(Image)`
   align-self: center;
   flex-shrink: 0;
-  width: ${props => props.theme.dimensions.categoryListItem.iconSize}px;
-  height: ${props => props.theme.dimensions.categoryListItem.iconSize}px;
-  margin: ${props => props.theme.dimensions.categoryListItem.margin}px;
+  width: ${props => props.theme.dimensions.native.categoryListItem.iconSize}px;
```

Comment:
Does this mean that it is possible to access the dimensions of the webapp within the native project?
I think it makes sense to only pass the configuration which is needed for the native project.

Reply:
See other answers